build/build-report.js

```diff
@@ -96,6 +96,11 @@ async function buildEsModulesBundle() {
     input: 'report/clients/bundle.js',
     plugins: [
       rollupPlugins.commonjs(),
+      // Exclude this 30kb from the devtools bundle for now.
+      rollupPlugins.shim({
+        [`${LH_ROOT}/shared/localization/i18n-module.js`]:
+            'export const swapLocale = _ => {}; export const format = _ => {};',
```

Comment:
Can you do what build-bundle does instead? https://github.com/GoogleChrome/lighthouse/blob/master/build/build-bundle.js#L112

Reply:
Oh, I guess you need some dummy functions exported here.